Xamarin.Forms.Controls/GalleryPages/SwipeViewGalleries/SwipeBehaviorOnInvokedGallery.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.GalleryPages.SwipeViewGalleries
+{
+	[Preserve(AllMembers = true)]
+	public class SwipeBehaviorOnInvokedGallery : ContentPage
+	{
+		public SwipeBehaviorOnInvokedGallery()
```

Comment:
If I'm understanding this gallery correctly it seems like the *Auto* isn't working here. It's acting the same as Close

Reply:
We are using SwipeItems with `Reveal` Mode. In Reveal mode, the SwipeView closes after an item is invoked. So, with `Auto` or `Close` values the SwipeView must close and with `RemainOpen` option must continue open.